src/main/resources/templates/springboot/eventhubs/build.eventhubs-client.gradle.tmpl

```diff
@@ -0,0 +1,9 @@
+ext.artifactId = "\${artifactId}-eventhubs-client"
+
+dependencies {
+    compile "com.blackbaud:common-eventhubs:1.+"
```

Comment:
Should this use a parameter for "commonEventHubsVersion"

Reply:
This is a separate build.gradle only for the client so this is the only place the version will be mentioned so its probably overkill. If you need to change it you just change it in this one spot